python/pyarrow/tests/test_dataset.py

```diff
@@ -472,6 +472,14 @@ def test_expression_boolean_operators():
         not true
 
 
+def test_expression_arithmetic_operators():
+    field = ds.field("field")
+    field + 1
+    field - 1
+    field * 1
+    field / 1
```

Comment:
This is a bit a dummy test until we can actually use those expressions (eg in https://github.com/apache/arrow/pull/9676)

Reply:
Now that #9676 is merged, do you want to update the test?

Reply:
ping @jorisvandenbossche

Reply:
Updated the test now